scripts/install

```diff
@@ -101,46 +102,50 @@ chown -R $app:www-data "$datadir"
 #=================================================
 # ADD A CONFIG FILE
 #=================================================
-ynh_script_progression --message="Adding a configuration file..."
+ynh_script_progression --message="Adding a configuration file…"
 
-duniter config --bma --ipv4 127.0.0.1 --port $port --remoteh $domain --remotep 443 --noupnp
-duniter config --ws2p-host 127.0.0.1 --ws2p-port 20901 --ws2p-remote-host $domain --ws2p-remote-port 443 --ws2p-remote-path "/ws2p" --ws2p-noupnp
+duniter --home $datadir config --bma --ipv4 127.0.0.1 --port $port --remoteh $domain --remotep 443 --noupnp
+duniter --home $datadir config --ws2p-host 127.0.0.1 --ws2p-port 20901 --ws2p-remote-host $domain --ws2p-remote-port 443 --ws2p-remote-path "/ws2p" --ws2p-noupnp
 
 #=================================================
 # SETUP SYSTEMD
 #=================================================
-ynh_script_progression --message="Configuring a systemd service..."
+ynh_script_progression --message="Configuring a systemd service…"
 
 # Create a dedicated systemd config
+sed -i "s@__DATADIR__@$datadir@g" ../conf/systemd.service
```

Comment:
should not be needed, as `ynh_add_systemd_config` use now `ynh_add_config` https://yunohost.org/fr/packaging_apps_helpers?q=%2Fpackaging_apps_helpers#ynh_add_config